examples/nuclio/ingest/ingest_example_test.go

```diff
@@ -1,9 +1,13 @@
+// +build nuclio_integration
```

Comment:
This one doesn't have a make target. Also, not really clear to me what this means... This integrates with a nuclio instance? Where does it find it? Or is this just integration against the sdk? If it's the latter, I don't see why we would want a tag for that. We use all kinds of libraries and don't mention it.